features/step_definitions/data-layer-migration.js

```diff
@@ -0,0 +1,22 @@
+import { Given, When, Then } from 'cucumber';
+
+const DATA_LAYER_MIGRATION_COMPONENT = '.DataLayerMigrationForm_component';
+
+Given(/^I haven't clicked the migration button$/, async function () {
+  await this.client.execute(() => {
+    daedalus.api.localStorage.reset();
```

Comment:
@daniloprates here we should probably use the `unsetDataLayerMigrationStart` call instead of complete reset